src/api-service/__app__/onefuzzlib/tasks/scheduler.py

```diff
@@ -102,7 +103,7 @@ def bucket_tasks(tasks: List[Task]) -> Dict[Tuple, List[Task]]:
 class BucketConfig(BaseModel):
     count: int
     reboot: bool
-    setup_url: str
+    setup_container: Container
```

Comment:
Does `Container` include the storage account?

Reply:
More to the point: do we lose important information by dropping the storage account, which is part of the SAS URL?

Reply:
Out-of-band answer: `Container` does not include the storage account, but we don't need this information, and we still correctly the recover the SAS URL when building the work set.